.github/workflows/ci.yml

```diff
@@ -7,17 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.x']
+        python-version: ["3.x"]
     name: Test by running cookiecutter with Python ${{ matrix.python-version }}
     env:
       TEST_PROJECT: foundation
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          path: 'orbyter-cookiecutter'
+          path: "orbyter-cookiecutter"
       - name: Setup python
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v4
```

Comment:
Are these version changes necessary? Not sure what the difference is. Are these just more up-to-date?

Reply:
The changes to [setup-python](https://github.com/marketplace/actions/setup-python) and [checkout](https://github.com/marketplace/actions/checkout) are not necessary except to suppress a Github warning about using Node12 instead of Node16.  As far as I can tell, they didn't introduce any breaking changes but have added flags, improved caching, and probably some bug fixes.